### ISSUE TITLE
Inconsistency between paper and code

### ISSUE THREAD

**Issue by wj-zhang** [liyunsheng13/BDL#26]
Hi, I find two inconsistencies between paper and code. Can you give me some suggestions? Thanks!

1. The paper says that "When training the segmentation adaptation model, images are resized with the long side to be 1,024 and the ratio is kept." However, for the dataloader, the code of image sizes is as follows
 https://github.com/liyunsheng13/BDL/blob/27dc56b3895619d0235c65f3f2c57b2a85fc98d6/data/__init__.py#L9

2. The paper says that "For FCN-8s with VGG16, we use Adam as the optimizer with momentum as 0.9 and 0.99. The initial learning rate is 1 × 10−5 and decreased with ‘step’ learning rate policy with step size as 5000 and γ = 0.1." However, the code for adjusting learning rate is as follows. I mean, is the number 5000 correct?
https://github.com/liyunsheng13/BDL/blob/27dc56b3895619d0235c65f3f2c57b2a85fc98d6/model/fcn8s.py#L188
https://github.com/liyunsheng13/BDL/blob/27dc56b3895619d0235c65f3f2c57b2a85fc98d6/model/discriminator.py#L43

**Comment by liyunsheng13** [comment]
For the image sizes, we just follow AdpatSegNet's work. (1024, 512) is only for the cityscapes datasets. 
For step size, I think there is a typo in the paper, it should be 50000. We will change it in our journal version. Thanks to point out.

**Comment by wj-zhang** [comment]
> For the image sizes, we just follow AdpatSegNet's work. (1024, 512) is only for the cityscapes datasets.
> For step size, I think there is a typo in the paper, it should be 50000. We will change it in our journal version. Thanks to point out.

Got it. Thank you so much for your help!
